pdp/static/js/pdp_map.js

```diff
@@ -47,8 +55,8 @@ function BC3005_map_options() {
     return options;
 }
 
-function BC3005_map_options_vic() {
-    var bounds = getBC3005Bounds_vic(),
+function BC3005_map_options_vic(archive_portal) {
+    var bounds = getBC3005Bounds_vic(archive_portal),
```

Comment:
It appears that the argument `archive_portal` isn't used in this function. Is it needed for some non-obvious reason?

Reply:
It's passed to `getBC3005Bounds_vic()` to determine the map bounds, a fact made unclear due to me naming the argument to `getBC3005Bounds_vic()`  something unrelated-seeming, instead of `archive_portal`, which is what this predicate is called everywhere else in the codebase. (https://github.com/pacificclimate/pdp/pull/156/files/a591efa2575be657d05c5b368181dfe1edb9d253#diff-708844e00533945241e16a8ef970f907R20). Thanks for catching that, I'll clear that up.